PresentScreenings/Entities/ScreeningsPlan.cs

```diff
@@ -26,7 +26,6 @@ public class ScreeningsPlan
         static string _filmInfoFile = Path.Combine(_directory, "filminfo.xml");
         List<Film> _films;
```

Comment:
All these lists can be static since the names of the files they are read from are static.
See also my comments on gathering access to these lists in one class, outside the main ViewController.
I propose to all define them as automatic properties.

Reply:
Gathering access to all static lists of `ScreeningsPlan` should be a separate user story.
Not changed.